{{cookiecutter.project_slug}}/postcss.config.js

```diff
@@ -0,0 +1,16 @@
+const purgecss = require('@fullhuman/postcss-purgecss')({
+  content: ['./{{ cookiecutter.project_slug }}/**/*.html'],
```

Comment:
We usually have another root folder, named `apps` that contains templates. But it's not in the template… It's still the case @sephii ?

Reply:
It's not the case anymore. Actually the `apps` folder is the named by the project name.